docs/UserGuide.md

```diff
@@ -103,64 +114,59 @@ Creates a new student record with the name of the student and other relevant det
+   All fields are compulsory and except for the tag field. A student can have multiple tags.
 </div>
 
 Examples:
 * `addstud nm/John Doe id/928C pn/Bob Doe hp/98765432 ad/311, Clementi Ave 2, #02-25 t/Peanut-Allergy t/No-Seafood`
 * `addstud nm/Alex Yeoh id/123A pn/Bernice Yu hp/99272758 ad/Blk 30 Lorong 3 Serangoon Gardens, #07-18`
 
-### 3.2 Viewing all student records : `viewAll`
-
-Shows a list of all students in the class.
-For each student in the list, only the Student's Name and Student's ID are displayed.
+### 4.1.2 Clearing all student records : `clear`
 
-:bulb: Note: The other details of a student are hidden to reduce cluttering.
-To view the full record of a student, use the `view` command instead.
+Clears all student records from local storage.
 
-Format: `viewAll`
+Format: `clear`
 
-### 3.3 Viewing student records from a class : `viewClass`
+### 4.1.3 Deleting a student record : `delete`
 
-Shows a list of all students in the specified class.
-
-Format: `viewClass [class name]`
+Deletes an existing student record from the class list, using the student’s name or the student’s ID.
 
-:bulb: Note: Class name is case-insensitive.
+Format: `delete nm/[name of student]` or `delete id/[id of student]`
 
 Examples:
-* `viewClass 2A`
-* `viewClass Loyalty1`
+* `delete nm/Jonathan Tan` deletes student record with student name as 'Jonathan Tan'.
+* `delete id/123A` deletes student record with student ID as '123A'.
 
 ### 3.4 Editing a student record : `edit`
 
 Edits the respective details of an existing student in the class list.
 
 Format: `edit [index] n/[name of student] id/[id of student] pn/[name of parent]…​`
 
-:bulb: Note:
-* Edits the person at the specified `index`. The index refers to the index number shown in the current displayed list. The index **must be a positive integer** 1, 2, 3, …​
-* Any tag can be used to edit the respective information.
-* At least one tag must be provided.
-* Existing values will be updated to the input values.
-
-Examples:
-*  `edit 1 pn/91234567 a/8 College Ave East` Edits the parent phone number and address of the 1st student to be `91234567` and `8 College Ave East` respectively.
-*  `edit 2 n/Jacob Teo` Edits the name of the 2nd student to be `Jacob Teo`.
-
-### 3.5 Deleting a student record : `delete`
-
-Deletes an existing student record from the class list, using the student’s name or the student’s ID.
-
-Format: `delete nm/[name of student]` or `delete id/[id of student]`
+<div markdown="span" class="alert alert-info">:information_source:
+**Note:** 
+   <ul>
+      <li> Edits the person at the specified `index`. The index refers to the index number shown in the current displayed list. The index **must be a positive integer** 1, 2, 3, …​ </li>
+      <li> Any tag can be used to edit the respective information. </li>
+      <li> At least one tag must be provided. </li>
+      <li> Existing values will be updated to the input values. </li>
+   </ul>
```

Comment:
I tried using HTML list tags previously but doesn't work 😢 . I believe you can use normal markdown syntax to list items.
```suggestion
- Edits the person at the specified `index`. The index refers to the index number shown in the current displayed list. The index **must be a positive integer** 1, 2, 3, …​
- Any tag can be used to edit the respective information.
```

Reply:
Okay, thanks daniel!